modules/rosa-planning-environment-application-reqs.adoc

```diff
@@ -0,0 +1,170 @@
+           tags: ''
+      objects:
+       - kind: DeploymentConfig
+         apiVersion: v1
+         metadata:
+           name: deploymentconfig${IDENTIFIER}
+         spec:
+           template:
+             metadata:
+               labels:
+                 name: replicationcontroller${IDENTIFIER}
+             spec:
+               enableServiceLinks: false
+               containers:
+               - name: pause${IDENTIFIER}
+                 image: "${IMAGE}"
+                 ports:
+                 - containerPort: 8080
+                   protocol: TCP
+                 env:
+                 - name: ENVVAR1_${IDENTIFIER}
+                   value: "${ENV_VALUE}"
+                 - name: ENVVAR2_${IDENTIFIER}
+                   value: "${ENV_VALUE}"
+                 - name: ENVVAR3_${IDENTIFIER}
+                   value: "${ENV_VALUE}"
+                 - name: ENVVAR4_${IDENTIFIER}
+                   value: "${ENV_VALUE}"
+                 resources: {}
+                 imagePullPolicy: IfNotPresent
+                 capabilities: {}
+                 securityContext:
+                   capabilities: {}
+                   privileged: false
+               restartPolicy: Always
+               serviceAccount: ''
+           replicas: 1
+           selector:
+             name: replicationcontroller${IDENTIFIER}
+           triggers:
+           - type: ConfigChange
+           strategy:
+             type: Rolling
+       - kind: Service
+         apiVersion: v1
+         metadata:
+           name: service${IDENTIFIER}
+         spec:
+           selector:
+             name: replicationcontroller${IDENTIFIER}
+           ports:
+           - name: serviceport${IDENTIFIER}
+             protocol: TCP
+             port: 80
+             targetPort: 8080
+           portalIP: ''
+           type: ClusterIP
+           sessionAffinity: None
+         status:
+           loadBalancer: {}
+       parameters:
+       - name: IDENTIFIER
+         description: Number to append to the name of resources
+         value: '1'
+         required: true
+       - name: IMAGE
+         description: Image to use for deploymentConfig
+         value: gcr.io/google-containers/pause-amd64:3.0
+         required: false
+       - name: ENV_VALUE
+         description: Value to use for environment variables
+         generate: expression
+         from: "[A-Za-z0-9]{255}"
+         required: false
+       labels:
+         template: deploymentConfigTemplate
+----
+
+The number of application pods that can run in a namespace is dependent on the number of services and the length of the service name when the environment variables are used for service discovery. `ARG_MAX` on the system defines the maximum argument length for a new process and it is set to `2097152 KiB` by default. The Kubelet injects environment variables in to each pod scheduled to run in the namespace including:
```

Comment:
I don't see a reason to enclose `2097152 KiB` in backticks.  If there's a guideline for that, please share.

I'm fairly confident that kublet is just lowercase.  I believe I can find some precedent in the Nodes section of the doc.